docs/pre.sh

```diff
@@ -27,9 +27,6 @@ cp ../depinject/README.md ./docs/packages/01-depinject.md
 cp ../collections/README.md ./docs/packages/02-collections.md
 cp ../orm/README.md ./docs/packages/03-orm.md
 
-## Add rosetta documentation
-cp ../tools/rosetta/README.md ./docs/run-node/04-rosetta.md
```

Comment:
If wget or curl is in the image that builds the docs, I would instead get `https://raw.githubusercontent.com/cosmos/rosetta/main/README.md` and leave the docs where they are.

Reply:
Yeah, I think we should just change that cp by a `wget -O` that gets rosetta docs from its repo. It is still useful for discoverability to have it displayed here: https://docs.cosmos.network/main/run-node/rosetta

Reply:
Done @julienrbrt